src/app/components/pages/Privacy.jsx

```diff
@@ -514,7 +514,7 @@ class Privacy extends React.Component {
                         <span className="c1" />
                     </p>
                     <p className="c2">
-                        <span className="c6">Personal Information</span>
+                        <h4 className="c6">Personal Information</h4>
```

Comment:
what are all these funny class names?

Reply:
Copy and pasted from the google-doc generated html.

Reply:
what is wrong with the TOS currently that this PR fixes? is this a style change?

Reply:
ah ic. ok. this makes more sense now :)